TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/UsbChangesRegister.java

```diff
@@ -0,0 +1,40 @@
+package com.novoda.tpbot.bot;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.hardware.usb.UsbManager;
+
+import static com.novoda.tpbot.bot.AndroidMovementService.ACTION_USB_PERMISSION;
+
+class UsbChangesRegister {
```

Comment:
This should really be an interface 🤔  because ATM we register for `Broadcasts` but this could feasibly change in the future.